features/step_definitions/developer_portal/admin/account/personal_details_steps.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+Given "the buyer wants to edit their personal details" do
+  visit admin_account_personal_details_path
+end
+
+When "the buyer edits their personal details" do
+  fill_in('user[username]', with: 'Alfred')
+  fill_in('user[email]', with: 'alfred@batcave.com')
+end
+
+And "they change their password" do
+  @new_password = 'ultrasecret'
+  fill_in('user[password]', with: @new_password)
+  fill_in('user[password_confirmation]', with: @new_password)
+end
+
+And "the buyer writes a wrong current password" do
+  fill_in('user[current_password]', with: 'megasecret')
+end
+
+And "the buyer writes a correct current password" do
+  fill_in('user[current_password]', with: 'supersecret')
```

Comment:
How do we know `supersecret` is the current password? Wouldn't it be more robust to get it from `@user`?